app/uk/gov/hmrc/transitmovementsrouter/controllers/MessagesController.scala

```diff
@@ -177,30 +173,64 @@ class MessagesController @Inject() (
       )
     ).toMovementAndMessageId
 
-  private def updateStatus(movementId: MovementId, messageId: MessageId, messageStatus: MessageStatus)(implicit
+  private def updateAndSendNotification(movementId: MovementId, messageId: MessageId, messageStatus: MessageStatus, jsonValue: JsValue)(implicit
     hc: HeaderCarrier
   ): EitherT[Future, PresentationError, Unit] =
-    persistenceConnector
-      .patchMessageStatus(movementId, messageId, MessageUpdate(messageStatus))
-      .asPresentation
+    for {
+      persistenceResponse <- persistenceConnector
+        .patchMessageStatus(movementId, messageId, MessageUpdate(messageStatus))
+        .asPresentation
+      _ = pushNotificationsConnector
+        .postJSON(
+          movementId,
+          messageId,
+          jsonValue
+        )
+    } yield persistenceResponse
 
   def handleSdesResponse() =
     Action.async(parse.json) {
       implicit request =>
         implicit val hc: HeaderCarrier = HeaderCarrierConverter.fromRequest(request)
-        (for {
-          sdesResponse <- parseAndLogSdesResponse(request.body)
-          (movementId, messageId) = extractMovementMessageId(sdesResponse)
-          persistenceResponse <- sdesResponse.notification match {
-            case SdesNotificationType.FileProcessed =>
-              updateStatus(movementId, messageId, MessageStatus.Success)
-            case SdesNotificationType.FileProcessingFailure =>
-              updateStatus(movementId, messageId, MessageStatus.Failed)
-            case _ => EitherT.rightT[Future, PresentationError]((): Unit)
-          }
-        } yield persistenceResponse).fold[Result](
-          error => Status(error.code.statusCode)(Json.toJson(error)),
-          _ => Ok
-        )
+        parseAndLogSdesResponse(request.body) match {
```

Comment:
Add a comment as to why this bit is separate please, just for easier understanding for someone else who comes in (i.e. why is `parseAndLogSdesResponse` separate?)